src/test/java/nextstep/courses/domain/SessionStateTest.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.courses.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SessionStateTest {
+    @Test
+    @DisplayName("강의 상태 준비중, 모집중, 종료가 아니면 예외가 던져진다")
+    void state_exception() {
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            SessionState.of("수강중");
+        });
+    }
+
+    @Test
+    @DisplayName("강의 상태 준비중, 모집중, 종료 3가지")
+    void create() {
+        SessionState sessionState1 = SessionState.of("모집중");
+        SessionState sessionState2 = SessionState.of("준비중");
+        SessionState sessionState3 = SessionState.of("종료");
+        assertThat(sessionState1).isEqualTo(SessionState.RECRUITING);
```

Comment:
```suggestion
        assertThat(SessionState.of("모집중")).isEqualTo(SessionState.RECRUITING);
```
굳이 로컬 변수로 구현하지 않아도 된다면 위와 같이 구현하는 것도 좋은 선택지가 될 수 있지 않을까?